crates/iota/src/iota_commands.rs

```diff
@@ -687,8 +700,13 @@ async fn start(
     // If this is set, then no data will be persisted between runs, and a new
     // genesis will be generated each run.
     if force_regenesis {
-        swarm_builder =
-            swarm_builder.committee_size(NonZeroUsize::new(DEFAULT_NUMBER_OF_AUTHORITIES).unwrap());
+        let committee_size = match committee_size {
+            Some(x) => NonZeroUsize::new(x),
+            None => NonZeroUsize::new(DEFAULT_NUMBER_OF_AUTHORITIES),
+        }
+        .ok_or_else(|| anyhow!("Committee size must be at least 1."))?;
```

Comment:
Should we use the clap `default_value_t` instead?

Reply:
No, we can't (I actually had the same thought as you, and had to change it back again). That's because we actually need to distinguish between the user providing it manually, or not. Provding it manually via flag should trigger a warning if the network.yaml is already present in the config dir. That wouldn't be possible if we just used a default instead.

Reply:
Alright, fair 👍